iroh/src/magicsock.rs

```diff
@@ -233,7 +235,7 @@ impl MagicSock {
     }
 
     fn is_closing(&self) -> bool {
-        self.closing.load(Ordering::Relaxed)
+        self.closing.load(Ordering::SeqCst)
```

Comment:
Someone smarter than me says that using `SeqCst` means you haven't really thought about what you're doing...